README.md

```diff
@@ -1,2 +1,5 @@
 # HTA-hub
 Website for a repository/inventory of health economics R packages 
+
+# Quarto render command
+quarto render
```

Comment:
````suggestion
To compile the website locally, run 
```
quarto render
```

This will generate static files in the `doc` folder. Open `docs/index.html` to view the website.
````